azurerm/internal/services/containers/container_registry_resource.go

```diff
@@ -170,6 +170,64 @@ func resourceArmContainerRegistry() *schema.Resource {
 				},
 			},
 
+			"policies": {
```

Comment:
I don't see the point of this block, could we pull out retention and trust policy to the top level for a better UX?

Reply:
was mostly trying to keep in line with the API, but yes happy to